R/constants.R

```diff
@@ -240,6 +240,12 @@ aglu.AVG_WOOD_DENSITY_KGM3 <- 500 # In kg per m3
 # Carbon content of wood is about 50 percent across species
 aglu.AVG_WOOD_DENSITY_KGCM3 <- 250 # In kg carbon per m3
 
+# Carbon content adjustments from unmanaged to managed
+aglu.CVEG_MULT_UNMGDFOR_MGDFOR <- 0.5
+aglu.CSOIL_MULT_UNMGDFOR_MGDFOR <- 0.87      #source: Guo and Gifford 2002
```

Comment:
Provide DOI if possible

Reply:
https://doi.org/10.1046/j.1354-1013.2002.00486.x

Reply:
Thanks, Page. @pralitp I'd suggest adding these clarifications to `constants.R`.